### ISSUE TITLE
How to pin messages

### ISSUE THREAD

**Issue by icarusinthewings** [oxen-io/session-pysogs#124]
Hello! Is it possible to make a pinned message containing, for example, community guidelines?
~ Judah

**Comment by jagerman** [comment]
Yes and no: From the SOGS perspective, yes, SOGS allows admins to pin/unpin multiple messages per room.  Session clients, however, have not yet implemented this functionality so that the moment it's kind of a latent feature.

**Comment by icarusinthewings** [comment]
> Yes and no: From the SOGS perspective, yes, SOGS allows admins to pin/unpin multiple messages per room. Session clients, however, have not yet implemented this functionality so that the moment it's kind of a latent feature.

Ah. Is it possible, then, to manipulate the database to pin a message?

**Comment by majestrate** [comment]
we could add a cli tool to do that, maybe?

**Comment by jagerman** [comment]
> Ah. Is it possible, then, to manipulate the database to pin a message?

Session clients aren't currently *reading* the pinned message status so it won't accomplish much.  I'll see if I can get the Session team to prioritize adding support for them, though, as it has been planned for a while.

**Comment by icarusinthewings** [comment]
> > Ah. Is it possible, then, to manipulate the database to pin a message?
> 
> Session clients aren't currently _reading_ the pinned message status so it won't accomplish much. I'll see if I can get the Session team to prioritize adding support for them, though, as it has been planned for a while.

In the session open group, there is a pinned message that appears. Is this an old version or something?

**Comment by ianmacd** [comment]
> > > Ah. Is it possible, then, to manipulate the database to pin a message?
> > 
> > 
> > Session clients aren't currently _reading_ the pinned message status so it won't accomplish much. I'll see if I can get the Session team to prioritize adding support for them, though, as it has been planned for a while.
> 
> In the session open group, there is a pinned message that appears. Is this an old version or something?

It appears only on Android. I believe it's hard-coded into the client.
